### ISSUE TITLE
Does not support Dark Mode

### ISSUE THREAD

**Issue by jrclooney** [sereneblue/chameleon#246]
Hi, I'm using the most current version of Chameleon and LOVE IT. But it doesn't work quite well. It has some interactions which causes it to possibly turn off my dark them in Windows. I am using Windows High Contrast dark theme in Windows 10 for instance and it causes visits to pages to turn to a white background. I want my screen to remain dark consistent with my theme. 

I think it also has something to do with NoScript with breaks it. 

I have disabled other plugins but may have tried it with 
Canvas Defender
Change Geolocation
Change Timezone
CyDec Platform Anti-Fingerprinting.
I also disabled Dark Mode
I have Dark Night Mode disabled
I have Dark Reader Disabled
Disable WebRTC
Disconnect
Facebook Container
HTTPS Everywhere
Privacy Badger
Random User-Agen
Toggle Referrer
uBlock Origin

So basically I want your plugin to not break my page or break my dark theme in Windows. Thanks. I am using the latest version of Firefox and Chameleon to the best of my knowledge.

**Comment by sereneblue** [comment]
Hi @jrclooney,

I don't use  the high contrast feature on Windows so I'm having a little difficulty following you. What exactly is Chameleon breaking? Does it break if you use a clean install (without any settings changed)? Can you post your exported settings and sites that Chameleon breaks for you?

**Comment by jrclooney** [comment]
Am closing ticket for now. Seems too many plugins were interacting with the plugin.
